health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/pages/employee/PlanInbox.js

```
@@ -539,6 +539,13 @@ const PlanInbox = () => {
       },
       sortable: true,
       width: "180px",
+      sortFunction: (rowA, rowB) => {
+        const fieldA = rowA?.[resource?.resourceType];
+        const fieldB = rowB?.[resource?.resourceType];
+        const valueA = parseFloat(fieldA || 0); 
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)